releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,15 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 3.0.0 - February 19, 2020
+* Added the following method to allow users to override `ConfigurationClientOptions`. These can be used to update the default retry options or configure a proxy for connecting to Azure App Configuration.
+
+   ````csharp
+   public AzureAppConfigurationOptions ConfigureClientOptions(Action<ConfigurationClientOptions> options)
+   ````
+* Added `IConfigurationRefresher.TryRefreshAsync` method to make it easier to trigger configuration refresh without crashing the application due to transient errors. Also, renamed the `Refresh` method to `RefreshAsync`.
```

Comment:
Can we put the rename into a separate bullet.

Reply:
The exception will not necessarily crash the app. It depends on how users call it. How about this?

"Added `IConfigurationRefresher.TryRefreshAsync` method, which will not throw exceptions on transient errors during configuration refresh. [Azure/AppConfiguration-DotnetProvider/issues/113]"

I agree to separate the rename one to make it more obvious.